vmServer/test/screenshot/code/execute.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/python
+"""Program to take screenshot from a MS Word Document
+"""
+
+import os
+import shutil
+import win32com.client
+import pythoncom
+import wx
+import win32con
+import repackage
+import win32gui
+repackage.up()
+
+
+
+
+def window_enumeration_handler(hwnd, top_windows):
+  """Handles window enumeration
+  """
+  top_windows.append((hwnd, win32gui.GetWindowText(hwnd)))
+
+
+def take_screenshot():
+  """Take screenshot from the MS word document
+  """
+  pythoncom.CoInitialize()
+  root_path = os.path.dirname(os.getcwd())
+  path_to_file = root_path+"\\accept\\execute\\action\\data\\sample.docx"
+  if os.path.exists(path_to_file):
+    word = win32com.client.Dispatch("Word.Application")
+    word.Visible = 1
+    word.Documents.Open(path_to_file)
+    top_windows = []
+    win32gui.EnumWindows(window_enumeration_handler, top_windows)
+    # Select the word document tab and bring it to foreground
+    # also maximise the word document
+    for i in top_windows:
```

Comment:
Prefer something more verbose than 'i', you can use 'window' if it suits here

Reply:
Variable name changed from i to windows